.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,28 @@
+<!--
+This form is for bug reports and feature requests ONLY!
+
+If you're looking for Kubernetes-specific help check [Stack Overflow](https://stackoverflow.com/questions/tagged/kubernetes)
+and the [troubleshooting guide](https://kubernetes.io/docs/tasks/debug-application-cluster/troubleshooting/).
+-->
+
+**Is this a BUG REPORT or FEATURE REQUEST?**:
+
+> Uncomment only one, leave it on its own line:
+>
+> /kind bug
+> /kind feature
+
+
+**What happened**:
+
+**What you expected to happen**:
+
+**How to reproduce it (as minimally and precisely as possible)**:
+
+
+**Anything else we need to know?**:
+
+**Environment**:
```

Comment:
Worth asking for stemcell version and bosh version? As we tend to push Bosh for stuff and stemcell xenial versions being out, could be worthy info.

Reply:
Stemcell for sure. Is there an easy way to get the version of the running director? `bosh -v` only seems to report the version of the CLI.

Reply:
It's worth adding the output of both `bosh -d <deployment> deployment` and `bosh -e <environment> env`. Good call.